### ISSUE TITLE
Add hpo xref to dementia

### ISSUE THREAD

**Issue by klaasMensaert** [monarch-initiative/mondo#6193]
**Mondo term (ID and Label):**
MONDO:0001627 - dementia

**Xref that should be fixed (ID and label):**
HP:0000726 - Dementia

**Your nano-attribution (ORCID)**
0000-0001-9859-8589

**Other comments:**
xref url: https://hpo.jax.org/app/browse/term/HP:0000726
(MONDO:otherHierarchy)

**Comment by nicolevasilevsky** [comment]
Hi @klaasMensaert - thanks for this ticket. We are focused on a [mass obsoletion](https://github.com/monarch-initiative/mondo/issues?q=is%3Aissue+is%3Aopen+label%3A%22mass+obsoletion%22) effort this month, and will look at this more closely in November or later. We appreciate your patience.

**Comment by nicolevasilevsky** [comment]
thanks @klaasMensaert, I added this xref
